devmgmtui/src/components/cloud/CloudDownload.js

```diff
@@ -176,6 +176,7 @@ class CloudDownload extends Component {
 				console.log(err);
 				
 				alert('Trouble fetching content from Diksha server.');
+				this.props.clearCurrentContent();
```

Comment:
This doesn't need to be handled under different conditions, like failed download etc..

Reply:
Not really. This is specific to search results and no other related events require clearing of the state.